tempesta_fw/tls.c

```diff
@@ -109,6 +110,7 @@ tfw_tls_msg_process(void *conn, struct sk_buff *skb, unsigned int off)
 		return TFW_PASS; /* more data needed */
 	} else if (r == 0) {
 		struct sk_buff *nskb;
+		TfwFsmData data = {};
```

Comment:
Just a small note. My understanding is that an empty initializer list is a C++ feature, and also a GNU C extension, but it's not in standard C syntax. There are samples of both `{}` and `{0}` (or `{ 0 }`) throughout Tempesta's code. Perhaps it's best to bring it to one common style.
https://stackoverflow.com/questions/17589533/is-an-empty-initializer-list-valid-c-code/17589839#17589839